components/arm/universalrobots/ur.go

```diff
@@ -462,7 +463,7 @@ func (ua *URArm) MoveToJointPositionRadians(ctx context.Context, radians []float
 
 		// TODO(erh): make responsive on new message
 		if !goutils.SelectContextOrWait(ctx, 10*time.Millisecond) {
```

Comment:
"SelectContextOrWait either terminates because the given context is done or the given duration elapses. It returns true if the duration elapsed."